gradle/libs.versions.toml

```diff
@@ -73,7 +73,9 @@ gradlePlugins-wire = { module = "com.squareup.wire:wire-gradle-plugin", version.
 guava = "com.google.guava:guava:32.1.2-jre"
 kotlinCliUtil = "com.slack.cli:kotlin-cli-util:2.1.0"
 kotlin-bom = { module = "org.jetbrains.kotlin:kotlin-bom", version.ref = "kotlin" }
+kotlin-compiler-embeddable = { module = "org.jetbrains.kotlin:kotlin-compiler-embeddable", version.ref = "kotlin"}
 kotlin-reflect = { module = "org.jetbrains.kotlin:kotlin-reflect", version.ref = "kotlin" }
+kotlin-stdlib = { module = "org.jetbrains.kotlin:kotlin-stdlib-jdk8", version.ref = "kotlin" }
```

Comment:
This is added automatically by the kotlin Gradle plugin, why did you have to add it?

Reply:
[2c1c508](https://github.com/slackhq/slack-gradle-plugin/pull/552/commits/2c1c5089c55de533b229ee1b1481390387b86a51)